src/emqx_protocol.erl

```diff
@@ -747,7 +747,11 @@ check_sub_acl(TopicFilters, PState) ->
       fun({Topic, SubOpts}, {Ok, Acc}) ->
               case emqx_access_control:check_acl(Credentials, subscribe, Topic) of
                   allow -> {Ok, [{Topic, SubOpts}|Acc]};
-                  deny  -> {error, [{Topic, SubOpts#{rc := ?RC_NOT_AUTHORIZED}}|Acc]}
+                  deny  ->
+                      emqx_logger:warning([{client, PState#pstate.client_id}],
+                                          "ACL(~s) Cannot SUBSCRIBE ~p for ACL Deny",
```

Comment:
`~s` for `Topic`?

Reply:
Yes, what's wrong?

Reply:
`~p` is wrong, it will print `<<"....">>`

Reply:
I supposed that it is better to print `<<"....">>` for topic filters.

Reply:
ok. fair enough.